apps/web/src/utils/auth-guards.ts

```diff
@@ -4,3 +4,7 @@ import { RoleTypes } from '@repo/db/types';
 export function isAdminOrModerator(session: Session | null) {
   return [RoleTypes.ADMIN, RoleTypes.MODERATOR].some((i) => session?.user.role.includes(i));
 }
+
+export function isAuthor(session: Session | null, userId?: string | null) {
```

Comment:
the return type here should be boolean. can you wrap in `Boolean`

Reply:
Sure